source/query-with-natural-language/ai-and-data-usage-information.txt

```diff
@@ -25,11 +25,18 @@ How Your Data is Used
 ---------------------
 
 When you query with natural language in Compass, the following 
-information is sent to MongoDB’s backend and/or the third party 
+information is sent to MongoDB's backend and/or the third party 
 AI provider: 
 
-The schema of the collection you are querying 
-(including collection name, field names, and types). 
+- The full text of your natural language prompt.
+- The full text of the AI-generated query.
```

Comment:
@Anemy do we send somewhere `the full text of the AI-generated query`? Do we log it or send to telemetry?

Reply:
cc @jaoppenheim